invalid clockid default in std.os.linux

### Zig Version

0.14.0-dev.591+1c35e73b6

### Steps to Reproduce and Observed Behavior

compile the project with the latest zig version.
```
https://github.com/andrewrk/poop/
cd poop
zig build. 
```
it errors out with 
```

std/os/linux.zig:7178:26: error: expected type 'os.linux.clockid_t', found 'comptime_int'
    clockid: clockid_t = 0,
                         ^
std/os/linux.zig:4058:23: note: enum declared here
pub const clockid_t = enum(u32) {
                      ^~~~
```

Changing `clockid:clockid_t=0` to `clockid:clockid_t=.REALTIME` fixes this. 

### Expected Behavior

Expected the program to compile and run.

--- a/lib/std/os/linux.zig
+++ b/lib/std/os/linux.zig
@@ -7175,7 +7175,7 @@ pub const perf_event_attr = extern struct {
     /// Defines size of the user stack to dump on samples.
     sample_stack_user: u32 = 0,
 
-    clockid: clockid_t = 0,
+    clockid: clockid_t = .REALTIME,
     /// Defines set of regs to dump for each sample
     /// state captured on:
     ///  - precise = 0: PMU interrupt

